### DISCUSSION TITLE
Cycling through tabs (CTRL+TAB) should exclude unloaded tabs + separate cycling of pinned and normal tabs

### DISCUSSION THREAD

**Discussion by stripedgoat** [zen-browser/desktop#2128] [Ideas]
When we cycle through tabs, it considers unloaded tabs as well.

I think it should exclude unloaded tabs.

It should only cycle through either the pinned tabs or normal tabs - depending on what's your currently active tab.

If you are on a pinned tab, then only cycle through the pinned tabs (excluding unloaded ones).

And the same way for normal tabs: If your active tab is a normal tab, the cycling applies only to normal tabs, also excluding unloaded ones.

**Comment by itaakesama** [comment]
I agree and now that there are essential and pinned tabs it is even more needed as cycling through tab is nightmare.

**Comment by No2515** [comment]
This has been an issue for me which is why I'm still using Sidebery
